.github/workflows/e2e.yml

```diff
@@ -0,0 +1,76 @@
+##
+# Run e2e tests
+##
+name: e2e
+on:
+  pull_request:
+
+jobs:
+  pre_build:
+    runs-on: macos-latest
+    steps:
+      - uses: fkirc/skip-duplicate-actions@master
+        with:
+          github_token: ${{ github.token }}
+          cancel_others: true
+
+  build:
```

Comment:
Let's call this `test` since that's the final outcome of this step.

Reply:
Have renamed